### PR TITLE
added `disabled` property

### PR DESCRIPTION
**Pull request by SterlingButters**
Drag disabling should now be supported by a callback

### CODE REVIEW

File: src/lib/components/dash_draggable.react.js
Comment (xhluca) [inline src/lib/components/dash_draggable.react.js:65]: Thanks for adding this. Have you tested whether the component is working correctly? I would recommend adding an usage example with a button that let you enable or disable the component.

File: .idea/dash-draggable.iml
Comment (xhluca) [inline .idea/dash-draggable.iml:5]: We should probably remove the entire `.idea/` folder. And add it to gitignore so it's not accidentally added.

File: dash_draggable/_imports_.py
Comment (xhluca) [inline dash_draggable/_imports_.py:4]: I don't think it's useful to modify this file, since it is automatically generated when you run `npm run build:py`.

File: usage.py
Comment (xhluca) [inline usage.py]: Probably better to keep it enabled by default. Otherwise people trying out usage.py without reading the source code will think it's broken.

**Review by xhluca — COMMENTED** [review]
Thanks for contributing @SterlingButters! Let me know if you are interested in supporting this component, I'd be happy to see more people improving this component.

File: src/lib/components/dash_draggable.react.js
Comment (SterlingButters) [reply src/lib/components/dash_draggable.react.js:65]: It does work correctly, I will try to add the `callback` to `usage.py` at some point. 

UPDATE: done - did with `BooleanSwitch`

File: .idea/dash-draggable.iml
Comment (SterlingButters) [reply .idea/dash-draggable.iml:5]: agreed, sorry, I forget about it... hmm already in my `.gitignore`? idk

File: usage.py
Comment (SterlingButters) [reply usage.py]: Fixed

**Comment by SterlingButters** [comment]
@xhlulu Once this PR is merged I will close #1

File: usage.py
Comment (xhluca) [inline usage.py:17]: Nice use of DAQ here! Could you add it to `requirements.txt` so that people using `pip install -r requirements.txt` could have it installed as well?

**Review by xhluca — APPROVED** [review]

**Comment by xhluca** [comment]
This is LGTM, once you made the change to requirements.txt i'll merge this

**Comment by xhluca** [comment]
Here's the link to be added as a collaborator, if you are interested! 

https://github.com/xhlulu/dash-draggable/invitations

File: usage.py
Comment (SterlingButters) [reply usage.py:17]: All done also added core components for kix if needed later

**Pull request merged**
